cluster-scope/base/core/configmaps/thanos-ruler-custom-rules/configmap.yaml

```diff
@@ -62,14 +62,10 @@ data:
         - alert: CustomCephStorageFillingUp
           annotations:
             summary: Ceph Storage is filling up.
-            description: "The Ceph Storage is {{ $value | humanizePercentage }} full."
-          expr: sum((kubelet_volume_stats_used_bytes{cluster="local-cluster"} * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info{cluster="local-cluster"}) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info{cluster="local-cluster"} * on (storageclass)  group_left(provisioner) kube_storageclass_info{cluster="local-cluster", provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"}))/sum((kube_persistentvolumeclaim_resource_requests_storage_bytes{cluster="local-cluster"} * on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info{cluster="local-cluster"}) * on (namespace,persistentvolumeclaim) group_left(storageclass, provisioner) (kube_persistentvolumeclaim_info{cluster="local-cluster"} * on (storageclass)  group_left(provisioner) kube_storageclass_info{cluster="local-cluster", provisioner=~"(.*rbd.csi.ceph.com)|(.*cephfs.csi.ceph.com)|(ceph.rook.io/block)"})) > 0.80
+            description: "The {{ $labels.pool }} pool Ceph Storage is {{ $value | humanizePercentage }} full."
+          expr: (ceph_pool_raw_used_bytes{pool=~"(nerc_ocp_prod_.*)|(nerc_ocp_infra_.*)"}) / on (cluster, namespace, pool) (ceph_pool_quota_max_bytes{pool=~"(nerc_ocp_prod_.*)|(nerc_ocp_infra_.*)"}) > 0.80
```

Comment:
would it make sense to test this trigger and alert by setting the ratio to somewhat low like 0.10 or whatever value that would trigger it? We can reset it back to what it should be afterwards.

Reply:
@naved001 you can now check out this `CustomCephStorageFillingUp` alert in `alerts-nerc-ocp` Slack chat now. I set it to 0.30 temporarily to make the alert show. 
![image](https://user-images.githubusercontent.com/15114160/231874458-e83f1d8d-8f7c-4dc8-94c8-69f74f56a2b6.png)